### ISSUE TITLE
Publish to package.elm-lang.org

### ISSUE THREAD

**Issue by johannhof** [blacktaxi/elm-random-secure#1]
Hey there, can you publish this library to http://package.elm-lang.org/? I'm also building a small toy password generator and would love not having to write my own WebCrypto bindings :)

Thanks!

**Comment by blacktaxi** [comment]
Hi there,

Last time I checked (see [here](https://groups.google.com/forum/#!searchin/elm-dev/native/elm-dev/1JW6wknkDIo/H9ZnS71BCAAJ)), there have been some problems with submitting native packages as it required a review from the package repository maintainers and they didn't have time, etc.

Would be happy to submit this to the official package repo, but if I'm not mistaken we still need to wait for the native package publishing process to be established.

**Comment by johannhof** [comment]
Huh, that's unfortunate. Thanks for the info! Should we leave this open as a reminder?

**Comment by blacktaxi** [comment]
Yes, let's keep it open. But JFYI, you can actually start using this now, though it requires some hacking around the package manager. You can check out an example of this in my other project, https://github.com/blacktaxi/pwdgen-elm.

**Comment by simnalamburt** [comment]
Hi I'm Hyeon Kim from the LibreIRC team.

We desperately needed this package but we found that this package was made with Elm 0.16. So we updated the whole codebases to Elm 0.18 and [made a PR](https://github.com/elm-lang/package.elm-lang.org/pull/223) to add it to the native whitelist.

@johannhof, you must have waited a long time for this. I hope evancz be positive with us. Please wish us a luck! Thanks!

###### References
- https://github.com/elm-lang/package.elm-lang.org/pull/223

**Comment by blacktaxi** [comment]
Hey @simnalamburt [elm-crypto](https://github.com/libreirc/elm-crypto) looks great!

Wonder why GitHub didn't register it as a proper fork?

**Comment by simnalamburt** [comment]
Ah... that was intentional.

At first, I tried to leave [elm-crypto](https://github.com/libreirc/elm-crypto) as a downstream fork of this project. But during the upgrade, I had to make too many breaking changes (i.e. https://github.com/libreirc/elm-crypto/issues/7) so I decided to diverge it from the upstream.

Since elm-random-secure is (maybe) license under BSD-3, your work is credited at the [COPYRIGHT](https://github.com/libreirc/elm-crypto/blob/master/COPYRIGHT#L18) file. I'm still going to introduce many breaking changes so I'm not sure if it can be merged into upstream. How do you think?
